toxcore/tox.h

```diff
@@ -551,6 +585,10 @@ bool tox_options_get_udp_enabled(const struct Tox_Options *options);
 
 void tox_options_set_udp_enabled(struct Tox_Options *options, bool udp_enabled);
 
+TOX_TRAVERSAL_TYPE tox_options_get_traversal_type(const struct Tox_Options *options);
```

Comment:
The declarations of these functions are generated by apidsl. There are some macros for creating the definitions as well. Can you add them? See the beginning of tox.c.